### PR TITLE
Gris 24.creating relationship between nodes

### PR DESCRIPTION
**Pull request by paulinaborys**
Implementation of relationship model, service and router.

### CODE REVIEW

File: graph_api/relationship/relationship_service.py [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+    Attributes:
+        database_url (str): URL to used database
+        database_auth (HTTPBasicAuth): Used to authenticate to database
+    """
+    database_url = (database["address"] + database["commit_path"]) \
+        .replace("{database_name}", database["name"])
+    database_auth = HTTPBasicAuth(database["user"], database["passwd"])
+
+    def save_relationship(self, relationship):
```
Comment (RioluM) [inline graph_api/relationship/relationship_service.py:19]: Based on #15 can be added
```suggestion
    def save_relationship(self, relationship: RelationshipIn):
```

File: graph_api/relationship/relationship_service.py [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+        if len(response['results'][0]['data']) == 1:
+            check_end_node_statement = f"MATCH (m) where id(m) ={relationship.end_node} return m"
+            commit_body = {
+                "statements": [{"statement": check_end_node_statement}]
+            }
+            response = requests.post(url=self.database_url,
+                                     json=commit_body,
+                                     auth=self.database_auth).json()
+
+            if len(response['results'][0]['data']) == 1:
```
Comment (RioluM) [inline graph_api/relationship/relationship_service.py:47]: Could be changed to function which takes node id as parameter and then used like that
```suggestion
        if node_exists(relationship.start_node) and node_exists(relationship.end_node):
        ...
```
then one 'else' statement below is not needed

File: graph_api/relationship/relationship_model.py [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+
+class RelationshipIn(BaseModel):
+    """
+        Model of relationship to acquire from client
+
+        Attributes:
+            start_node (int): Id of node which starts connection
+            end_node: (int): Id of node which ends connection
+            name
```
Comment (mrwrob) [inline graph_api/relationship/relationship_model.py:12]: Missing description

File: graph_api/relationship/relationship_service.py (hunk 2 above)
Comment (mrwrob) [reply graph_api/relationship/relationship_service.py:47]: That's good idea :+1: These multiply elses don't look good.

**Review by mrwrob — CHANGES_REQUESTED** [review]

**Comment by paulinaborys** [comment]
@mrwrob I've done all your requests

**Comment by mrwrob** [comment]
Nice job, thank you :confetti_ball:

**Pull request merged**